docs/markdown/Reference-manual.md

```diff
@@ -104,7 +104,10 @@ Add a custom test setup that can be used to run the tests with a
 custom setup, for example under Valgrind. The keyword arguments are
 the following:
 
-- `env` an [environment object](#environment-object) to use a custom environment
+- `env` environment variables to set, such as `['NAME1=value1',
+  'NAME2=value2']`, a dictionary *(since 0.52.0)*, or an
+  [`environment()` object](#environment-object) which allows more
+  sophisticated environment juggling
```

Comment:
I would add the new addition in the end, otherwise IMHO it's not clear of environment() part is since 0.52 or not.

What about:
```
`env` environment variables to set, such as `['NAME1=value1',
  'NAME2=value2']`, or an [`environment()` object](#environment-object)
  which allows more sophisticated environment juggling. *Since 0.52.0* a dictionary is also accepted.```

Reply:
wfm